packages/forms-web-app/src/views/manage-appeals/your-appeals.njk

```diff
@@ -42,7 +42,6 @@
                   </td>
                   <td class="govuk-table__cell govuk-table__cell--numeric">
 										{% if item.displayInvalid %}
```

Comment:
should we remove this and shuffle the elif into the first if?

Reply:
I had kept it there because we have a default 'else' at the end of the if block, which will display the dueInDays figure and we want this column to remain blank (otherwise it would show 'days').  Might be able to reword that final else block.

Reply:
ah ok, can we get some sort of comment to that effect then please